### PR TITLE
dummy release

### PR DESCRIPTION
**Pull request by prateekbh**

### CODE REVIEW

**Comment by artipost** [comment]
Failed pictures (1): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/6d3babfb12bcdfdb1106a12dfe84512c/raw/2a979d6d81ee5fc40cb6db6b4b88ddeba89e5e19/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [449493081](https://travis-ci.org/prateekbh/preact-material-components/builds/449493081), for commit [a2a9207](https://github.com/prateekbh/preact-material-components/pull/1145/commits/a2a92077202ee6823648f4bb76b4437612420f13).

File: package.json [hunk 1]
```diff
@@ -75,7 +75,8 @@
     "@material/top-app-bar": "0.39.3",
     "@material/typography": "0.39.0",
     "bind-decorator": "^1.0.11",
-    "material-components-web": "0.39.3"
+    "material-components-web": "0.39.3",
+    "serve": "^7.0.0"
```
Comment (cromefire) [inline package.json]: Does this have to be in the root `package.json`?

If not consider moving it to `docs/package.json`

Comment (prateekbh) [reply package.json]: great point

**Comment by cromefire** [comment]
And please look at the image and replace it

**Comment by prateekbh** [comment]
replace with what?

**Comment by cromefire** [comment]
Sorry you have already done it

**Comment by cromefire** [comment]
Travis error:
`Error: Cannot find module 'serve'`
Serve > 6 does not have a JavaScript API, I suggest using the old version, it's replaced in `2.0` anyway.

**Comment by artipost** [comment]
Failed pictures (1): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/7e8bd224975c4a7ee34ae9ee86213ee5/raw/0bf3abffba205abb56d5eef7f64167884b113125/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [449538699](https://travis-ci.org/prateekbh/preact-material-components/builds/449538699), for commit [af23908](https://github.com/prateekbh/preact-material-components/pull/1145/commits/af23908ba3239c284152c2d6675cf4cb262b63d2).

**Comment by artipost** [comment]
Failed pictures (1): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/009f7bcdfc968df3aa4bbe4dbd71e6cc/raw/417d7a09be6b6c3717516a8a73e5d7a001be89d6/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [449543963](https://travis-ci.org/prateekbh/preact-material-components/builds/449543963), for commit [27f8fd9](https://github.com/prateekbh/preact-material-components/pull/1145/commits/27f8fd940383f286dabef018c7d28e71f89fbed1).

**Comment by prateekbh** [comment]
ahhh!!! whats the problem with this thing 😢

**Comment by artipost** [comment]
Failed pictures (1): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/c7fe037f0d3a72aab5b11fa26513e42f/raw/a070b598c05423f47a56ec803320c2333b3e8dc7/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [449550156](https://travis-ci.org/prateekbh/preact-material-components/builds/449550156), for commit [1334856](https://github.com/prateekbh/preact-material-components/pull/1145/commits/133485647d709c3bb3e81c32a31a547dbfdc52e1).

**Comment by artipost** [comment]
Failed pictures (1): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/5779c79181489fe5b0eeebb342f6423b/raw/6e6e7318a436e86a5c03cbd3769330381a68c43f/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [449553707](https://travis-ci.org/prateekbh/preact-material-components/builds/449553707), for commit [dc8c4a7](https://github.com/prateekbh/preact-material-components/pull/1145/commits/dc8c4a7b16260ce18748be7fb4950100a6947f09).

**Comment by artipost** [comment]
Failed pictures (25): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/a6f0f0ffd74e4a71f8f678546d7e856f/raw/09cd63245e90f7529258259d779eaccfb7c7a73e/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [449557622](https://travis-ci.org/prateekbh/preact-material-components/builds/449557622), for commit [a3b60b9](https://github.com/prateekbh/preact-material-components/pull/1145/commits/a3b60b932281747e90fca65252eb849300fd70cc).

**Comment by artipost** [comment]
Failed pictures (2): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/3bbac38af119d43da863fc77d4d8375c/raw/b59dc37dc22a8afde7191dc0fb8b91ae858eba94/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [449561994](https://travis-ci.org/prateekbh/preact-material-components/builds/449561994), for commit [f8cf779](https://github.com/prateekbh/preact-material-components/pull/1145/commits/f8cf779463ceda424648599bcb75c76885c8f59e).

**Comment by artipost** [comment]
Failed pictures (1): [failed-pictures.zip](https://gist.githubusercontent.com/artipost/ab3e2d8d7398875ccf457240b6ed17ca/raw/3626c61db115b06e7f4fd39e993bb4c962085ceb/failed-pictures.zip)

##
:postbox: Posted by [artipost.io](https://artipost.io), through Travis build [449566047](https://travis-ci.org/prateekbh/preact-material-components/builds/449566047), for commit [bb59f27](https://github.com/prateekbh/preact-material-components/pull/1145/commits/bb59f271ae9275baff2509eb029beadbf8ccaf0f).

**Comment by cromefire** [comment]
I'm still thinking of a better solution to test the that, but besides comparing the Dom, I didn't have any idea. I still thing that could do it.

**Comment by prateekbh** [comment]
the only problem with dom comparison is that it does not test if we screw up css

**Comment by cromefire** [comment]
Right, they are both kind of bad...

**Comment by cromefire** [comment]
We would need something that is serializing the elements and their css rules, but I don't think there is any way to access the computed css rules, is there?

**Comment by prateekbh** [comment]
there's something `computedStyleMap`

**Comment by cromefire** [comment]
> In Firefox, Opera and Safari you can use window.getComputedStyle(element)

Well I have an idea how to do that if you have nothing against it.

**Comment by cromefire** [comment]
We could have a result like:
```json
{
    "type": "div",
    "attributes": {
        […]
    },
    style: {
        […]
    }
}
```

**Comment by cromefire** [comment]
https://codepen.io/anon/pen/KrPzmg
Should we filter which one are compared? And if so whitelist or blacklist?

**Comment by cromefire** [comment]
As we are using Firefox for testing in `2.0`, we can use `Window.getDefaultComputedStyle()` to filter only the changed values.

**Pull request merged**